installer/install.iss

```diff
@@ -383,8 +383,7 @@ const
 
     // Git Credential Manager settings.
     GCM_None          = 1;
-    GCM_Classic       = 2;
-    GCM_Core          = 3;
```

Comment:
Just to check, these values are local only to the installer, and not stored or persisted to the registry/install-options?

Reply:
Yes, these are just an internal index for the array of radio buttons. The persisted values are human readable strings, not numeric values.